ui/scripts/bootstrap.sh

```diff
@@ -51,7 +51,7 @@ elif [[ ! -f $CHAOS_DASHBOARD_BIN/chaos-dashboard ]]; then
 
   cd ..
   IN_DOCKER=1 make images/chaos-dashboard/bin/chaos-dashboard
-  cd -
+  cd ... || return
```

Comment:
As this is the last line of the script, maybe we could remove this `cd`? @g1eny0ung 

`cd ... || return` looks like an invalid bash script (`cd` doesn't accept `...`).

Reply:
Sorry I missed this comment. I agree with you. this line can be removed. @anuragpaliwal80 Please help update it when you have time? Thx!